client/tests/integration/components/character-counter-test.js

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | character-counter', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`<CharacterCounter />`);
+
+    assert.equal(this.element.textContent.trim(), '');
+
+    // Template block usage:
+    await render(hbs`
+      <CharacterCounter>
+        template block text
+      </CharacterCounter>
+    `);
+
+    assert.equal(this.element.textContent.trim(), 'template block text');
```

Comment:
aw, c'mon. it'll take 1 minute to add a test?

Reply:
😇

Reply:
I added three. 😎